testutils/src/main/java/com/amplifyframework/testutils/sync/SynchronousDataStore.java

```diff
@@ -13,14 +13,15 @@
  * permissions and limitations under the License.
  */
 
-package com.amplifyframework.testutils;
+package com.amplifyframework.testutils.sync;
 
 import androidx.annotation.NonNull;
 
 import com.amplifyframework.core.Amplify;
 import com.amplifyframework.core.model.Model;
 import com.amplifyframework.datastore.DataStoreException;
 import com.amplifyframework.datastore.DataStoreItemChange;
+import com.amplifyframework.testutils.Await;
```

Comment:
If you have a sync package, this should probably go there too, since that’s what this util is all about

Reply:
I considered `Await` as almost a primitive tool used by the the items inside sync package, so I didn't want to group it in there. I feel like things like `Await`, `Latch`, and `Sleep` all basic building blocks of test utilities, so they should probably be just stay where they are. Wasn't sure what to do with the rest.